src/components/FullHeader/FullHeader.js

```diff
@@ -93,9 +94,11 @@ const FullHeader = ({
 					</Menu.Item>
 				) : null}
 
-				<Menu.Item key="/apps">
-					<Link to="/apps">Apps</Link>
-				</Menu.Item>
+				{apps && Boolean(Object.keys(apps).length) && (
```

Comment:
@jyash97 I think if no apps exists, then we won't even be showing this page. i.e. new users will only have the option to create a cluster going forward.

Reply:
Users will be able to access the route by directly going to `/apps`.